packages/core/lib/generators/assets_generator.dart

```diff
@@ -99,14 +102,20 @@ String generateAssets(
 List<String> _getAssetRelativePathList(
   String rootPath,
   List<String> assets,
+  List<String> excludes,
 ) {
   final assetRelativePathList = <String>[];
```

Comment:
How about filtering the result of assetRelativePathList after the `for` expression.
Example:
```dart
final assetRelativePathList = <String>[];
for (final assetsName in assets) {
...
}
return assetRelativePathList.where(...).toList();
```